interpreter/src/main/scala/pl/touk/nussknacker/engine/definition/TypeInfos.scala

```diff
@@ -75,12 +75,23 @@ object TypeInfos {
     }
   }
 
-
-  case class ClazzDefinition(clazzName: TypedClass,
+  case class ClazzDefinition(clazzName: TypingResult,
```

Comment:
Let's add tests in `InterpreterSpec` showing that it is possible to pass anything into `Unknown` and it is possible to use it inside the fragment